docs/data_access.rst

```diff
@@ -0,0 +1,218 @@
+    v0.4.0 we are will distribute the SQLite databases and Apache Parquet files
+    alongside a set of data packages. As of PUDL v0.5.0 we will be generating SQLite
+    and Apache Parquet outputs directly, and will no longer be archiving tabular data
+    packages as the format of record, and the format conversions described below will
+    no longer be necessary.
+
+Archived Data Packages
+^^^^^^^^^^^^^^^^^^^^^^
+
+We periodically publish data packages containing the full outputs from the PUDL ETL
+pipeline on `Zenodo <https://zenodo.org>`__, and open data archiving service provided
+by CERN. The most recent release can always be found through this concept DOI:
+`10.5281/zenodo.3653158 <https://doi.org/10.5281/zenodo.3653158>`__. Each individual
+version of the data releases will be assigned its own unique DOI.
+
+All of our archived products can be found in the `Catalyst Cooperative Community on
+Zenodo <https://zenodo.org/communities/catalyst-cooperative/>`__. These archives and
+the DOIs associated with them should be permanently accessible, and are suitable for
+use as references in academic and other publications.
+
+Once you've downloaded or generated your own tabular data packages you will probably
+want to convert them into a more analysis oriented file format. We typically use
+SQLite for the core FERC and EIA data, and Apache Parquet files for the very long
+tables like EPA CEMS.
+
+Converting to SQLite
+^^^^^^^^^^^^^^^^^^^^
+
+If you want to access the data via SQL, we have provided a script that loads several
+data packages into a local :mod:`sqlite3` database. Note that these data packages
+**must** have all been generated by the **same** ETL run, or they will be considered
+incompatible by the script. For example, to load three data packages generated by our
+example ETL configuration into your local SQLite DB, you could run the following
+command from within your PUDL workspace:
+
+.. code-block:: console
+
+    $ datapkg_to_sqlite \
+        datapkg/pudl-example/ferc1-example/datapackage.json \
+        datapkg/pudl-example/eia-example/datapackage.json \
+
+Run ``datapkg_to_sqlite --help`` for more details.
+
+Converting to Apache Parquet
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :doc:`data_sources/epacems` data approaches 100 GB in size uncompressed, which is
+too large to work with directly in memory on most systems, and take a very long time
+to load into SQLite. Instead, we recommend converting the Hourly Emissions table into
+an `Apache Parquet <https://parquet.apache.org>`__ dataset which is stored on disk
+locally, and either reading in only parts of it using pandas, or using `Dask
+dataframes <https://dask.org>`__, to serialize or distribute your analysis tasks.
+Dask can also speed up processing for in-memory tasks, especially if you have a
+powerful system with multiple cores, a solid state disk, and plenty of memory.
+
+If you have generated an EPA CEMS data package, you can use the
```

Comment:
This part might not be clear to a user. How does one generate a data package? They can also be downloaded from Zenodo, no?